franklin/config/urls.py

```diff
@@ -18,13 +18,17 @@
 
 from .views import health
 
-from github.views import auth, callback, deploy_hook, register_repo
+from github.views import deploy_hook, register_repo
+from users.views import UserLogin
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^auth/', auth, name='auth'),
-    url(r'^callback/', callback, name='callback'),
+    url(r'^$', UserLogin.as_view()),
```

Comment:
@JulianGindi Need to talk about the auth and callback methods in github views. Not sure if they are relevant anymore.